samples/fabricaConfig-1org-1channel-1chaincode.json

```diff
@@ -61,7 +61,15 @@
       "channel": "channel1",
       "init": "{\"Args\":[]}",
       "endorsement": "AND ('Org1MSP.member')",
-      "directory": "./chaincodes/chaincode-kv-node"
+      "directory": "./chaincodes/chaincode-kv-node",
+      "privateData": [
```

Comment:
We copy that file to user during `fabrica init` so it should be as concise and simple as possible.